src/main/java/com/google/devtools/build/lib/query2/common/CommonQueryOptions.java

```diff
@@ -253,4 +270,30 @@ public AspectResolutionModeConverter() {
               + "precise mode is not completely precise: the decision whether to compute an aspect "
               + "is decided in the analysis phase, which is not run during 'bazel query'.")
   public AspectResolver.Mode aspectDeps;
+
+  ///////////////////////////////////////////////////////////
+  // GRAPH OUTPUT FORMATTER OPTIONS                        //
```

Comment:
Are the proto:<foo> options above also graph output formatter options?

Reply:
Not to my knowledge? As far as I can see those only apply to `--output=proto`

Reply:
Ah, I didn't quite read this as --output=graph output formatter options. makes sense.